contrib/babelfishpg_tsql/src/pltsql_ruleutils.c

```diff
@@ -377,6 +379,82 @@ tsql_get_constraintdef(PG_FUNCTION_ARGS)
 	PG_RETURN_TEXT_P(string_to_text(res));
 }
 
+PG_FUNCTION_INFO_V1(tsql_get_expr);
+
+/* ----------
+ * pg_get_expr          - Decompile an expression tree
```

Comment:
Update the comment. Rename pg_get_expr to tsql_get_expr.

Reply:
Changes Done.